cpp/unreal_plugins/CoreUtils/Source/CoreUtils/CoreUtils.Build.cs

```diff
@@ -17,6 +17,7 @@ public CoreUtils(ReadOnlyTargetRules Target) : base(Target)
         PCHUsage = ModuleRules.PCHUsageMode.Default;
         PrivatePCHHeaderFile = "";
         bUseUnity = false;
+        bEnableUndefinedIdentifierWarnings = false;
```

Comment:
Put this after bEnableExceptions and include a comment explaining why it's here. We have worked hard to document all of our mystery Unreal settings (e.g., in all INI and CS files). I want to avoid introducing new mystery settings without documenting them.

Reply:
I added a comment as to why this param is required.